python/pyarrow/table.pxi

```diff
@@ -4708,6 +4711,61 @@ cdef class Table(_PandasConvertible):
 
         return table
 
+    def drop(self, columns):
+        """
+        Drop one or more columns and return a new table.
+
+        Alias of Table.drop_columns for backwards compatibility.
+
+        Parameters
```

Comment:
Thanks! I tried this at first, but thought I was getting a document build error. I probably was just building it wrong 😬 haha. I'll do this because I very much dislike duplication.